olp-cpp-sdk-dataservice-read/src/generated/api/BlobApi.cpp

```diff
@@ -20,6 +20,8 @@
 #include "BlobApi.h"
 
 #include <map>
+#include <memory>
```

Comment:
Leftover from previous patch-set

Reply:
nevertheless, it's required for shared_ptr used in the file.